### DISCUSSION TITLE
Composer require oddvalue/laravel-drafts:1.5.2 for Laravel 11 not resolving properly

### DISCUSSION THREAD

**Discussion by vinodsptharsha** [oddvalue/laravel-drafts#56] [Q&A] [answered]
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - laravel/framework is locked to version v11.10.0 and an update of this package was not requested.
    - carbonphp/carbon-doctrine-types 3.2.0 conflicts with doctrine/dbal 3.8.4.
    - oddvalue/laravel-drafts v1.5.2 requires doctrine/dbal ^3.3 -> satisfiable by doctrine/dbal[3.3.0, ..., 3.8.4].
    - nesbot/carbon 3.5.0 requires carbonphp/carbon-doctrine-types * -> satisfiable by carbonphp/carbon-doctrine-types[3.2.0].
    - symfony/http-foundation v7.1.1 conflicts with doctrine/dbal 3.5.4.
    - symfony/http-foundation v7.1.1 conflicts with doctrine/dbal 3.4.6.
    - symfony/http-foundation v7.1.1 conflicts with doctrine/dbal 3.3.8.
    - symfony/http-foundation v7.1.1 conflicts with doctrine/dbal 3.3.5.
    - laravel/framework v11.10.0 requires nesbot/carbon ^2.72.2|^3.0 -> satisfiable by nesbot/carbon[3.5.0].
    - laravel/framework v11.10.0 requires symfony/http-foundation ^7.0 -> satisfiable by symfony/http-foundation[v7.1.1].
    - Root composer.json requires oddvalue/laravel-drafts 1.5.2 -> satisfiable by oddvalue/laravel-drafts[v1.5.2].

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

**Comment by oddvalue** [comment, marked as answer]
I've just release v2 that will install properly in a Laravel 11 project.
